### PR TITLE
Remove old error toasts

### PR DESCRIPTION
**Pull request by cakecatz**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/web/src/constants/toast-text.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
 export const UPDATE_STATIC_ADMIN_INFO_SUCCESS = `Successfully updated Static Admin configurations.`;
 export const UPDATE_STATIC_ADMIN_INFO_FAILED = `Failed to update Static Admin configurations.`;
```
Comment (nghialv) [inline pkg/app/web/src/constants/toast-text.ts:2]: This can be removed too.

File: pkg/app/web/src/constants/toast-text.ts [hunk 2]
```diff
@@ -1,6 +1,5 @@
...
 export const UPDATE_RBAC_SUCCESS = `Successfully updated RBAC configurations.`;
-export const UPDATE_RBAC_FAILED = `Failed to update RBAC configurations.`;
 export const UPDATE_SSO_SUCCESS = `Successfully updated SSO configurations.`;
 export const UPDATE_SSO_FAILED = `Failed to update SSO configurations.`;
```
Comment (nghialv) [inline pkg/app/web/src/constants/toast-text.ts:5]: This one too.

File: pkg/app/web/src/constants/toast-text.ts (hunk 1 above)
Comment (cakecatz) [reply pkg/app/web/src/constants/toast-text.ts]: Thank you!

**Pull request merged**